Clarify zoom levels and dev usage (#72)
hubmapconsortium/vitessce-image-viewer: README.md

For the demo, run `npm start` and you will be able to update the component and use the
`demo/src/App.js` to visually test.

For development,
HTTP is acceptable but potentially slower than HTTP2 for `TIFF`. However, for

> development with `zarr`, you will likely want to use an [HTTP2 server](https://github.com/GoogleChromeLabs/simplehttp2server)

Reviewer: ```suggestion
development with `zarr`, you should use an [HTTP2 server](https://github.com/GoogleChromeLabs/simplehttp2server)
```

> although you are welcome to use HTTP, again. You'll see that our demo

Reviewer: ```suggestion
for best performance. Our demo
```

> is hooked up to Google Cloud, which is HTTP2 by default.

Reviewer: This might be better in the IMAGE_RENDERING.md. We need to be more definite.

Author: I think this is important info so I'd like to keep this on the front page.  I think Bruce was confused about this initially.

> is hooked up to Google Cloud, which is HTTP2 by default.

Reviewer: ```suggestion
uses Google Cloud Storage, which is HTTP2 by default.
```

##### `initialViewState` (object) [ONLY NECESSARY FOR `MicrsocopyViewer`]

An object containing two things

- `target` (Array) An `[x,y,0]` location in image coordinates of the image. The 0
  represents a hypothetical (and potentially future addition) of a third spatial dimension.
- `zoom` (Number) The initial zoom level to render the image at

##### `minZoom` & `maxZoom` (Number)

These control the max and min zoom sizes, generally the number of images `n` in your pyramid,
ranging from `-n` (`minZoom`), the lowest resolution, to `0` (`maxZoom`), the highest resolution. You don't need to
pass these in (they can be inferred from heigh/width/tile size), although

> if you wish to limit zoom levels, you may.

Reviewer: What happens if they are set?: Does it actually stop someone from zooming father out or in? When would someone want them?

Author: Yes, that is the use case.  We could not expose it - the `zarr` setting can infer it as well.
